spec/features/staff/users_can_create_a_user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.feature "Creating a user" do
+  let(:user) { build(:user) }
```

Comment:
How about renaming this `user` to something like `target_user`, to differentiate between the admin user who is creating the user *AND* the user being created?